scss/_core.scss

```diff
@@ -17,3 +17,4 @@
 
 //Utiliticore/es
 @import "core/tokens/_sizing.scss";
+@import "core/tokens/_disable.scss";
```

Comment:
Do we need a separate files for every tokens. I think this can go in one page.

Reply:
https://www.lightningdesignsystem.com/design-tokens/

Reply:
Ya sure, will make the changes

Reply:
We have separate CSS files for all token classes. 
![Screenshot from 2020-05-11 16-18-10](https://user-images.githubusercontent.com/34312966/81553675-251f9300-93a3-11ea-8e4d-9dc452d336b9.png)
I think we can combine all tokens in on MDX file and keep separate CSS files?

Reply:
we can use `utilities.scs`s for small tokens like, cursor-pointer, disabled etc.. These small classes do not deserve to have separate files.

Reply:
Ok, sure